multiqc/modules/dragen/mapping_metrics.py

```diff
@@ -0,0 +1,447 @@
+
+        # flattening phenotype-sample data by adding a prefix " normal" to the normal samples
+        data_by_sample = dict()
+        for sn in data_by_phenotype_by_sample:
+            for phenotype in data_by_phenotype_by_sample[sn]:
+                new_sn = sn
+                if phenotype == 'normal':
+                    new_sn = sn + '_normal'
+                data_by_sample[new_sn] = data_by_phenotype_by_sample[sn][phenotype]
+
+        if not data_by_rg_by_sample and not data_by_phenotype_by_sample:
+            return
+        log.info("Found mapping metrics for {} samples".format(len(data_by_rg_by_sample)))
+
+        self.report_mapping_metrics(data_by_sample, data_by_rg_by_sample)
+
+    def report_mapping_metrics(self, data_by_sample, data_by_rg_by_sample):
+        # merging all read group data
+        data_by_rg = dict()
+        for sname in data_by_rg_by_sample:
+            for rg, d in data_by_rg_by_sample[sname].items():
+                if rg in data_by_rg:
+                    rg = rg + ' (' + sname + ')'
+                data_by_rg[rg] = d
+
+        # getting all available metric names to determine table headers
+        all_metric_names = set()
+        for sn, d_by_rg in data_by_rg_by_sample.items():
+            for rg, data in d_by_rg.items():
+                for m in data.keys():
+                    all_metric_names.add(m)
+        # and making headers
+        # headers, beeswarm_keys = make_mapping_stats_headers(all_metric_names)
+        genstats_headers, own_tabl_headers = make_headers(all_metric_names, MAPPING_METRICS)
+
+        self.general_stats_addcols(data_by_sample, genstats_headers, namespace=NAMESPACE)
+
+        self.add_section(
+            name='Mapping metrics per RG',
+            anchor='dragen-mapping-metrics',
+            description='Mapping metrics, similar to the metrics computed by the samtools-stats command. '
+                        'Shown on per read group level. To see per-sample level metrics, refer to the general '
+                        'stats table.',
+            plot=table.plot(data_by_rg, own_tabl_headers, pconfig={'namespace': NAMESPACE})
+        )
+
+        # Make bargraph plots of mapped, dupped and paired reads
+        self.__map_dup_read_chart(data_by_rg)
+        self.__map_pair_read_chart(data_by_rg)
+
+        # Alternatively can consider a beeswarm plot, similar to samtools flagstats:
+        # self.add_section(
+        #     name='Mapping metrics per RG',
+        #     anchor='dragen-mapping-metrics',
+        #     description="A dot plot showing DRAGEN mapping metrics for each input read group. "
+        #                 "All read counts in " + str(config.read_count_desc) + ", " +
+        #                 "all bases counts in " + str(config.read_count_desc),
+        #     plot=beeswarm.plot(data_by_rg, beeswarm_keys, {'id': 'dragen-mapping-metrics-dp'})
+        # )
```

Comment:
Can remove this I think. Or, maybe better, disable it by default and look for a module-specific config option to turn it on ([docs](https://multiqc.info/docs/#user-configuration)).